010-Getting-started/010-what-is-xata.mdx

```diff
@@ -1,300 +1,119 @@
+### Dedicated Postgres clusters
 
-## Going global
+<Alert status="warning">Dedicated clusters are currently in private alpha. Please reach out to request access.</Alert>
 
-As we’ve seen above, we call the combination of PostgreSQL + Elasticsearch and the associated services a “cell”. The cells are grouped into regions and form our **data plane**.
+These are single-tenant clusters that are dedicated to a single customer. You can create multiple Xata databases or branches on a dedicated cluster, and you can move them between clusters (including shared clusters) without downtime.
 
-![Xata global regions](/images/global-regions.png)
+Unlike the shared clusters, the dedicated clusters offer the full set of PostgreSQL functionality, including custom functions and triggers. They have no rate limits and no concurrency limits, and are limited only by the compute size.
 
-Besides the cells, the regions contain other services:
+Xata dedicated clusters separate storage from compute and the storage and compute are billed separately for dedicated clusters. The storage grows automatically and you only get billed for what you use. In addition, the storage pricing is significantly cheaper than the storage pricing for shared clusters. The compute is billed by the hour depending on the instance size and characteristics.
 
-- An API Gateway service exists in each region and is responsible for routing between cells and enforcing the rate limits.
-- Several other supporting regional services, including regional control plane services and a service for running long-running tasks.
-- A global control plane layer responsible for maintaining users, workspaces, and API keys.
+The above means that the dedicated clusters are typically cost-efficient for applications with larger database sizes and higher traffic.
 
-The control plane information (users, workspaces, API keys) is stored in a single central region and then replicated in near real-time to all other regions. This way, we can offer minimal latencies in each geographical area while still maintaining the convenience of a global service.
+There are two kind of instance types to be used in dedicated clusters:
 
-## Data model representation
+- **Provisioned**, which are more cost effective for applications with predictable workloads.
+- **Auto-scaling**, which are more cost effective for applications with unpredictable workloads.
 
-Xata has a relational data model, with a strict schema (schemaful) and with support for JSON-like objects. Records are grouped into tables, which are grouped into databases. Xata supports rich column types and relations between tables can be represented via link columns, which are similar to foreign keys.
+The dedicated clusters, both provisioned and auto-scaling, are powered by AWS Aurora. The auto-scaling functionality is provided by AWS Aurora Serverless v2.
 
-A record in Xata can look something like this:
+## More than PostgreSQL
```

Comment:
If we end up having one large section for "postgres on Xata" then we can probably ditch this header / section and jump straight into the secondary data store.

Reply:
Agreed with this and applied.